code/game/objects/items.dm

```diff
@@ -345,6 +345,29 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 
 			else if(S.can_be_inserted(src))
 				S.handle_item_insertion(src)
+		return 
+	if (istype(user,/mob/living/carbon/human))

```

Comment:
human is the type that has the personal crafting datum attached to it that i use below.